<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyDlg</class>
 <widget class="QDialog" name="CopyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>正在拷贝</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbTotal">
     <property name="text">
      <string>正在计算……</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbTotal">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbSource_2">
       <property name="text">
        <string>从：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSource">
       <property name="text">
        <string>源</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbTarget_2">
       <property name="text">
        <string>到：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTarget">
       <property name="text">
        <string>目标</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbCurItem">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CopyDlg</receiver>
   <slot>sltCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltCancel()</slot>
 </slots>
</ui>
